package.json

```diff
@@ -2,6 +2,7 @@
   "name": "fh-fhc",
   "description": "A Command Line Interface for FeedHenry",
   "version": "2.0.0-BUILD-NUMBER",
+  "minPlatformVersion": "7.15.0-1661",
```

Comment:
Might be worth adding an underscore

http://stackoverflow.com/questions/10065564/add-custom-metadata-or-config-to-package-json-is-it-valid. 

Not sure where else I could put it. It's own file maybe?